xarray/tests/test_units.py

```diff
@@ -915,24 +958,39 @@ def test_merge_dataarray(variant, unit, error, dtype):
     )
 
     func = function(xr.merge)
```

Comment:
Is this wrapping with `function` needed here?

Reply:
no, I think not. I'll remove it.